content/en/security_chaos_engineering.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Security Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Security Chaos Engineering or SCE is a discipline based on [Chaos Engineering](/chaos_engineering/). SCE performs proactive security experimentation on a distributed system to build confidence in the system's capability to withstand turbulent and malicious conditions. Security chaos engineers use scientific method loops to achieve this, including steady-state, hypothesis, continuous verification, lesson learned, and mitigation implementation.
+
+## Problem it addresses
+The main priority for [site reliability engineers](/site_reliability_engineering/) (SREs) and cyber security engineers is to restore service as fast as possible with the goal of achieving zero downtime and minimizing business impact. Incidents and practices deal both with pre-failure and post-failure situations. Most security issues are challenging to discover and patch quickly, impacting application or system functionality. Additionally, security incidents are usually tricky to uncover during the development phase.
```

Comment:
I'm struggling to understand how these sentences connect. The first sentence is clear, but how does "incidents and practices deal both with pre-failure and post-failure situations" fit in and how does it connect to the following sentence. In the last sentence, I added "additionally" because it feels like an extra point. 

I did do some edits but don't think I changed the meaning of the sentences (although I may have by mistake).

Reply:
you are right, let me rephrase it, adding "SREs and Cyber Security Engineers" as subject